.github/workflows/build.yml

```diff
@@ -456,7 +456,7 @@ jobs:
           @echo on
           python --version
           python -m pip install -U wheel setuptools
-          python setup.py --data bazel-bin -q bdist_wheel --nightly $BUILD_NUMBER
+          python setup.py --data bazel-bin -q bdist_wheel --nightly %BUILD_NUMBER%
```

Comment:
I am not familiar with this. What kind of syntax is this?

Reply:
@terrytangyuan `%ENV%` is the Windows CMD way of specifying environmental variable (since we are building on windows platform).

Reply:
Got it. Thanks!

Reply:
@terrytangyuan It is quite old age... here is the documentation:
https://support.microsoft.com/en-us/help/121170/how-to-access-environment-variables-in-an-ms-dos-batch-file